net/sslh/Makefile

```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=        sslh
+PLUGIN_VERSION=     0.1
+PLUGIN_DEVEL=       yes
+PLUGIN_COMMENT=     sslh configuration front-end
```

Comment:
most plugins are frontends. the opportunity here is to describe the software being configured in a nutshell :)

Reply:
I'll update the comment to describe the sslh service.